homeassistant/components/axis/__init__.py

```diff
@@ -64,3 +78,19 @@
     }
 
     hass.config_entries.async_update_entry(config_entry, options=options)
+
+
+async def async_migrate_entry(hass, config_entry):
+    """Migrate old entry."""
+    LOGGER.debug("Migrating from version %s", config_entry.version)
+
+    if config_entry.version == 1:
+        device = config_entry.data.pop(CONF_DEVICE)
+        config_entry.data = {**config_entry.data, **device}
+
+        config_entry.version = 2
+        hass.config_entries.async_update_entry(config_entry)
```

Comment:
No need for this. Happens when this function returns True.

Reply:
Fixed